sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Release History
 
-## 1.4.0 (Unreleased)
+## 1.4.0 (2023-08-24)
 
 ### Features Added
 
-- Added inline image support
-- Updated @azure/communication-signaling to 1.0.0-beta.22 with inline image support
+- Added `ChatAttachment` object that contains properties for inline images incoming from Teams Chat Threads.
+- Updated to @azure/communication-signaling@1.0.0-beta.22 with inline image support
```

Comment:
```suggestion
- Updated @azure/communication-signaling to 1.0.0-beta.22 with inline image support
```

Reply:
the original wording is fine to me, why did we update it?

Reply:
I update this to following previous `Changelog` format 

## 1.3.1 (2023-02-28)

### Features Added

- Updated to `@azure/communication-common` 2.2.0.
- Updated to @azure/communication-signaling@1.0.0-beta.16. Added GCCM real-time notification support.